src/test/regress/Makefile

```diff
@@ -124,6 +124,14 @@ check-multi-non-adaptive: all
 	--server-option=citus.task_executor_type=real-time \
 	-- $(MULTI_REGRESS_OPTS) --schedule=$(citus_abs_srcdir)/multi_schedule $(EXTRA_TESTS)
 
+check-multi-hyperscale: all
+	$(pg_regress_multi_check) --constr="$(constr)" --hoststr="$(hoststr)" --load-extension=citus \
```

Comment:
psql's documentation (https://www.postgresql.org/docs/current/app-psql.html) calls connection string parameter `conninfo`, maybe we should also do the same.

Reply:
If we expect `hoststr` to always contain two hostnames, then why not replace it with two parameters `--worker_1_public_hostname` and `--worker_2_public_hostname`?